src/System.IO.FileSystem/tests/FileStream/ctor_sfh_fa.cs

```diff
@@ -17,17 +15,33 @@ protected virtual FileStream CreateFileStream(SafeFileHandle handle, FileAccess
         }
 
         [Fact]
-        public void InvalidHandleThrows()
+        public void InvalidHandle_Throws()
         {
-            AssertExtensions.Throws<ArgumentException>("handle", () => CreateFileStream(new SafeFileHandle(new IntPtr(-1), true), FileAccess.Read));
+            using (var handle = new SafeFileHandle(new IntPtr(-1), ownsHandle: false))
+            {
+                AssertExtensions.Throws<ArgumentException>("handle", () => CreateFileStream(handle, FileAccess.Read));
```

Comment:
We get that for things that actually create a "real" handle. What I don't want here is SafeFileHandle calling the OS to release the handle. Granted, in this case, it's moot, because the handle is invalid. I kept it here to show intent and be consistent- this is a "fake" test handle.